_includes/footer.html

```diff
@@ -40,199 +65,69 @@
-                        Protection Bureau<br>
-                        1700 G Street, NW<br>
-                        Washington, D.C. 20552
-                    </p>
-                    <div class="footer_general-inqueries">
-                        <h4 class="footer_general-inqueries-label">
-                            <i>For general inquiries only</i>
-                        </h4>
-                        <ul class="list list__links">
-                            <li class="list_item">
-                                <a class="list_link footer_general-inqueries-phone" href="tel:202-435-7000">
-                                    (202) 435-7000
-                                </a>
-                            </li>
-                            <li class="list_item">
-                                <a class="list_link" href="mailto:info@consumerfinance.gov">
-                                    info@consumerfinance.gov
-                                </a>
-                            </li>
-                        </ul>
-                    </div>
+            <div class="footer-middle-left">
 
-                </div><!-- END .footer_nested-col-group -->
-            </div><!-- END .footer-left -->
-
-        </div><!-- END .footer_nested-col-group -->
+                <div class="footer_col">
+                    <ul class="footer_list">
+                        <li class="list_item">
+                            <a class="list_link list_link__disabled">
+                                TBD Content
+                            </a>
+                        </li>
+                        <li class="list_item">
+                            <a class="list_link list_link__disabled">
+                                TBD Content
+                            </a>
+                        </li>
+                        <li class="list_item">
+                            <a class="list_link list_link__disabled">
+                                TBD Content
+                            </a>
+                        </li>
+                        <li class="list_item">
+                            <a class="list_link list_link__disabled">
+                                TBD Content
+                            </a>
+                        </li>
+                    </ul>
+                </div>
+                <div class="footer_col">
+                    <ul class="footer_list">
+                        <li class="list_item">
+                            <a class="list_link list_link__disabled">
+                                TBD Content
+                            </a>
+                        </li>
+                        <li class="list_item">
+                            <a class="list_link list_link__disabled">
+                                TBD Content
+                            </a>
+                        </li>
+                    </ul>
+                </div>
+            </div>
 
+        <div class="footer-middle-right">
```

Comment:
The block above was a little over-indented. I went through and cleaned that up. Thanks for the catch.